plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -260,6 +265,20 @@ export async function createRouter(
         'Cache-Control': 'no-cache',
         'Content-Type': 'text/event-stream',
       });
+      let shouldUnsubscribe = false;
+
+      // Send heartbeat every 2 seconds so the frontend/proxy knows the connection is still alive
+      const timeout = setTimeout(() => {
```

Comment:
should this be interval?

Reply:
The name of the variable?

Reply:
is this called in a loop? setTimeout executes the passed in function once after two seconds, setInterval would execute it every two seconds. The code comment doesn't match the logic here

Reply:
Ah ok. Thank you!